dom/src/test/java/org/apache/james/mime4j/message/MessageBuilderTest.java

```diff
@@ -46,7 +47,7 @@
 
     @Test
     public void testSetCustomBody() throws Exception {
-        MessageBuilder builder = MessageBuilder.create();
+        MessageBuilder builder = MessageBuilder.of();
```

Comment:
Do you think I should duplicate all for create and of?
Now that create is deprecated I do not think it worst testing it that much than before?
I don't know what would you do ?